src/components/application_manager/src/state_controller_impl.cc

```diff
@@ -88,7 +88,10 @@ void StateControllerImpl::SetRegularState(ApplicationSharedPtr app,
     return;
   }
 
-  if (app->is_resuming() && !IsResumptionAllowed(app, state)) {
+  const bool app_is_resuming = app->is_resuming();
```

Comment:
Extract it to separate commit if possible

Reply:
@LuxoftAKutsan It is already in the separate commit - 8b8216822ccc953dadf09ca51aae928e9f8edd01